mobilecoind/src/config.rs

```diff
@@ -203,8 +209,6 @@ impl Config {
 
         let conn = GrpcFogReportConnection::new(self.peers_config.chain_id.to_owned(), env, logger);
 
-        let verifier = self.get_fog_ingest_verifier();
```

Comment:
Sorry for my ignorance, but was there a reason to move this line?

Reply:
It resolved a borrow checker issue, we could also have cloned the logger